### PR TITLE
[FIX] Improve decommission process_2

### PR DESCRIPTION
**Pull request by McSim85**
Often, when I run the role with `state: absent`, the next role run with `state: present` will fail with different errors. For example, like described in the bug
https://github.com/artis3n/ansible-role-tailscale/issues/429

After talking to Tailscale support, that issue might happen if `/var/lib/tailscale/` still contains the previous machine identity (state, auth key, id, etc). 

We have to remove this folder during decommission.
Fortunately, Ubuntu package has that task inside the `tailscale.postrm`, (if the user rune `apt purge tailscale`):
```sh
$ cat /var/lib/dpkg/info/tailscale.postrm
#!/bin/sh
set -e
if [ -d /run/systemd/system ] ; then
	systemctl --system daemon-reload >/dev/null || true
fi

if [ -x "/usr/bin/deb-systemd-helper" ]; then
    if [ "$1" = "remove" ]; then
		deb-systemd-helper mask 'tailscaled.service' >/dev/null || true
	fi

    if [ "$1" = "purge" ]; then
		deb-systemd-helper purge 'tailscaled.service' >/dev/null || true
		deb-systemd-helper unmask 'tailscaled.service' >/dev/null || true
		rm -rf /var/lib/tailscale
	fi
fi
```
So, this is the best way to handle tailscale state removal - `purge: true,` because this will always be aligned with the package maintainers config.

I am not sure about other distros, though.
Probably, we need a task like this in each distro subtree:
```yaml
- name: Uninstall | Delete Tailscale Package State
  ansible.builtin.file:
    path: "/var/lib/tailscale"
    state: absent
```
@artis3n , 
As we discussed here
https://github.com/artis3n/ansible-role-tailscale/pull/430#discussion_r1493393305
I removed variable from the code. 
Please let me know how do you prefer to handle this or if you need any help?

P.S. This is part of 
https://github.com/artis3n/ansible-role-tailscale/pull/430

### CODE REVIEW

**Comment by McSim85** [comment]
Missed molecule assertion. 
Will push PR shortly.

File: tasks/uninstall.yml [hunk 1]
```diff
@@ -56,3 +56,8 @@
 - name: Uninstall | OpenSUSE
   when: ansible_distribution in tailscale_opensuse_family_distros
   ansible.builtin.include_tasks: opensuse/uninstall.yml
+
+- name: Uninstall | Destroy Tailscale Package State
+  ansible.builtin.file:
+    path: "/var/lib/tailscale"
```
Comment (artis3n) [inline tasks/uninstall.yml:62]: According to <https://tailscale.com/kb/1069/uninstall#completely-delete-tailscale>, please change this (and the `state-absent` verification) to `/var/lib/tailscale/tailscaled.state`.

**Review by artis3n — CHANGES_REQUESTED** [review]

File: tasks/uninstall.yml (hunk 1 above)
Comment (McSim85) [reply tasks/uninstall.yml]: done in [2883a5f](https://github.com/artis3n/ansible-role-tailscale/pull/435/commits/2883a5fa5d4f9a080d791b2effcb207b43b66cd8)

File: tasks/uninstall.yml [hunk 2]
```diff
@@ -56,3 +56,8 @@
 - name: Uninstall | OpenSUSE
   when: ansible_distribution in tailscale_opensuse_family_distros
   ansible.builtin.include_tasks: opensuse/uninstall.yml
+
+- name: Uninstall | Destroy Tailscale Package State
+  ansible.builtin.file:
```
[the remark this answers is not in the document]
Comment (McSim85) [reply tasks/uninstall.yml]: How did I miss that 🫣
Thanks for reviewing 🙏

**Review by artis3n — APPROVED** [review]
Thank you for this PR!

**Pull request merged**